tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/XUnit/V3/XUnitTestAssemblyRunnerRunV3Integration.cs

```diff
@@ -43,7 +43,7 @@ internal static CallTargetState OnMethodBegin<TTarget, TContext>(TTarget instanc
             var testBundleString = new AssemblyName(assemblyName).Name ?? string.Empty;
 
             // Extract the version of the framework from the TestClassRunner base class
-            var frameworkType = instance.GetType();
+            var frameworkType = instance!.GetType();
```

Comment:
Haha, I guess this is a consequence of me suggesting removing the `instance` check 😅 can _probably_ fix it with it a `where TTarget : notnull`, but whether we want/should do that is a different question 😅 I still think you _should_ have a `context.Instance is null` check, but I don't know...